src/color-slider/README.md

```diff
@@ -188,6 +195,13 @@ If you’re only using the CSS file, you should set these yourself.
 | `valuechange` | Fired when the value changes for any reason, and once during initialization. |
 | `colorchange` | Fired when the color changes for any reason, and once during initialization. |
 
+### Parts
+
+| Name | Description |
+|------|-------------|
+| `slider` | The internal `<input type="range">` element. |
+| `spinner` | The internal `<input type="number">` element inside the `tooltip` slot. |
```

Comment:
```suggestion
| `spinner` | The default `tooltip` slot content (an `<input type="number">` element). Please note that if an element is slotted in the `tooltip` slot, this will not match anyhing. |
```